essay.tex

```diff
@@ -0,0 +1,61 @@
+
+\begin{document}
+
+\maketitle
+
+\abstract The Agile development methodology is the most prominent within the software development. It is at it's core designed to be Agile and to be adaptable for any and every team following it's guidelines. However that can also lead teams failing to fully utilise the Agile methodology. This paper will explore how the Agile method can impact the motivation of teams and the impact it has had on some existing teams.
+
+\section{Introduction}
+Motivation as described in the Oxford dictionary 1.1, is the ``Desire or willingness to do something; enthusiasm:''. It is this enthusiasm, the desire to work, that I would like to address.
+
+In this paper the definiton of team will be the one suggested by Hackman, but further expanded. \cite{GroupExec} To simplify it, it  states that ``A team is a collection of individuals who are interdependent in their tasks, who share responsibility for outcomes'' 
+
+This paper will outline the motivational impacts of the Agile methodology \cite{Agile} within a team by using case studies and other research that has been conducted around this topic. This paper will be excluding motivational factors like career prospects and job security where possible, though this are also areas to consider for any team they should be excluded for the sake of maintaining the integrity of the paper. 
+
+\section{The research}
+One of the central idea's suggested by Cockburn and Highsmith \cite{AgilePeople} is that Agile helps to improve motivation and morale, by implementing increased communication and reduced documentation. This helps to build a sense of community within the team which then in turn improves their morale.
+
+Looking at the work of Asproni \cite{MotivationSoftware} he explores the key motivational factors for software developers. Drawing the conclusion that a developer is more interested in the work being interesting, than being paid more. It's not difficult to assume that people working within the games industry being interested in their work is a huge benefit and will help to improve their enthusiasm for the project. 
+
+Research carried out by McHugh, Conboy and Lang \cite {AgilePractices} shows some interesting results in which they spoke with already established teams, allowing them skip over as Tuckman \cite {Tuckman} suggests in his work the initial stages of Forming, Norming, etc. Reducing the possible effects on the teams motivations that can occur. In their research they found that many of the people who had experience from a more traditional management style, had little desire to work on a team that did not use an Agile approach.
+ 
+The research from various authors shows that the key to productive teams is: to make sure you build a sense of community, to make sure that the team are involved in work they find interesting and to let them experience other types of management so they can decide which is best.
+
+\section{Agile in the Gaming Industry}
```

Comment:
Relegating the games industry aspect of the essay to a mere section is a poor strategy. The discussion should be at the fore of the essay in the opening, referred to throughout the argument, and then contextualized in the conclusion. Though games and game development are referenced here and there, aspire to a more integrated approach. Bring it into the foreground.